critiquebrainz/frontend/views/review.py

```diff
@@ -58,6 +59,8 @@ def browse():
                            page=page, limit=limit, count=count, entity_type=entity_type)
 
 
+# TODO : Refactor this function to remove PyLint warning.
```

Comment:
In CB code we put an identifier next to a TODO comment to know who to ask if more details are needed. So something like
```python
# TODO(psolanki): Refactor this function to remove PyLint warning.
```